src/puppetlabs/puppetdb/http/query.clj

```diff
@@ -179,6 +175,19 @@
   (restrict-query ["=" "title" title]
                   req))
 
+(defn wrap-in-from
```

Comment:
fwiw I expected to see this function use an in clause and a from clause until seeing the implementation.